core/helpers/index.ts

```diff
@@ -171,3 +171,11 @@ export const processURLAddress = (url: string = '') => {
   if (url.startsWith('/')) return `${config.api.url}${url}`
   return url
 }
+
+export function bottomVisible () {
```

Comment:
as it would be a helper now we should add some check if this was not invoked by server and return some default value